### DISCUSSION TITLE
Welcome to Learn-for-Everything Discussions!

### DISCUSSION THREAD

**Discussion by UNLINEARITY** [UNLINEARITY/Learn-Everything#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bill-safe** [comment]
作者是否有考虑利用LLM大语言模型或者一些RAG项目将这些资料整合做成一个AI问答系统

**Reply by UNLINEARITY** [reply]
还真没考虑过，作者目前大三😴，目前主要还是在累计的阶段，一点点手敲上去的。之所以目前没有很好借助 AI，主要还是觉得大语言模型还是不够精准，生成的答案还是有随机性，而且回答的结果范围过于广泛而粗略。我其实是想着多找些人来搞不同领域的专业知识，毕竟只有身在一个学习的领域内，对问题的把握才更加精准而有深度。可惜目前主要还是一人在搞，自己想要精准而系统的愿望也难以实现了，而且对于 AI 的时代而言，也确实做的工作很“慢”且“有限”了。所以现在更多的是将自己学习的部分内容上传以供自己思考和学习巩固了。还是想扩展的，只是太难了。
